syntax = "proto3";

service Showtime {
  // Get all Times of broadcast of movies from database
  rpc GetListTimes(EmptyMsg) returns (stream MoviesSchedules) {};
  // Get a Movie by its broadcast date
  rpc GetMovieByDate (Date) returns (Movies);
}

message Date {
  string date = 1;
}

message Movies {
  repeated string movies = 1;
}

message MoviesSchedules {
  string date = 1;
  repeated string movies = 2;
}

message EmptyMsg {
}